ravenframework/MessageHandler.py

```diff
@@ -92,28 +80,33 @@ def __init__(self):
     self.suppressErrs = False
     self.printTime    = True
     self.inColor      = False
-    self.verbCode     = {'silent':0, 'quiet':1, 'all':2, 'debug':3}
-    self.colorDict    = {'debug':'yellow', 'message':'neutral', 'warning':'magenta', 'error':'red'}
-    self.colors={
-      'neutral' : '\033[0m',
-      'red'     : '\033[31m',
-      'green'   : '\033[32m',
-      'yellow'  : '\033[33m',
-      'blue'    : '\033[34m',
-      'magenta' : '\033[35m',
-      'cyan'    : '\033[36m'}
-    self.warnings     = [] #collection of warnings that were raised during this run
-    self.warningCount = [] #count of the collections of warning above
+    self.verbCode     = {'silent' : 0,
+                         'quiet'  : 1,
+                         'all'    : 2,
+                         'debug'  : 3}
+    self.colorDict    = {'debug'  : 'yellow',
+                         'message': 'neutral',
+                         'warning': 'magenta',
+                         'error'  : 'red'}
+    self.colors       = {'neutral': '\033[0m',
+                         'red'    : '\033[31m',
+                         'green'  : '\033[32m',
+                         'yellow' : '\033[33m',
+                         'blue'   : '\033[34m',
+                         'magenta': '\033[35m',
+                         'cyan'   : '\033[36m'}
+    self.warnings     = [] # collection of warnings that were raised during this run
+    self.warningCount = [] # count of the collections of warning above
 
   def initialize(self, initDict):
     """
       Initializes basic instance attributes
       @ In, initDict, dict, dictionary of global options
       @ Out, None
     """
-    self.verbosity = initDict.get('verbosity','all').lower()
-    self.callerLength = initDict.get('callerLength',25)
-    self.tagLength = initDict.get('tagLength',15)
+    self.verbosity    = initDict.get('verbosity', 'all').lower()
+    self.callerLength = initDict.get('callerLength', 25)
+    self.tagLength    = initDict.get('tagLength', 15)
```

Comment:
aligning "=" isn't expressly forbidden, but it is non-pep8. In generally we've been removing sign alignments and favoring comment alignments.

Reply:
Yeah, this is stuff that Andrea held over from Fortran; it was an ongoing debate on whether we should or not. I think we ended up going with pep8's extremely strong opinion on this one in the end.